conu/backend/docker/image.py

```diff
@@ -238,19 +238,29 @@ def _run_container(self, run_command_instance, callback):
             container_id = fd.read()
         return container_id, response
 
-    def run_via_binary(self, run_command_instance=None, *args, **kwargs):
+    def run_via_binary(self, command=None, additional_opts=None, *args, **kwargs):
         """
         create a container using this image and run it in background;
         this method is useful to test real user scenarios when users invoke containers using
         binary
 
-        :param run_command_instance: instance of DockerRunBuilder
+        :param command: list of str, command to run in the container, examples:
```

Comment:
I vote we'll be backwards compatible and allow the first arg to be instance of `DockerRunBuilder`.

We'll break backwards compat on renaming from run_command_instance to command, but I'm +1 on that.

Reply:
I'd suggest leaving `run_command_instance` for `DockerRunBuilder` and adding `command` as another argument. Using both of them might result in an Exception.

Reply:
Do we reorder? will command be first?

Reply:
I think it would be beneficial not to reorder. There will be no confusion when someone writes e.g `i.run_via_binary(['--name', 'jana'], ['echo', 'jana'])`, what results in strange error, not easily caught by conu.

Reply:
I agree!

So the result is, just to make it clear:
1. add `run_command_instance` back as the first argument with the former functionality
2. make `command` the second argument with proposed functionality

Reply:
I agree, but what about the tests? Now they are all changed to comply with newly proposed functionality. Do we want to changed some of them back to be sure about backwards compatibility?

Reply:
We should definitely have at least one test to verify both code paths, for the rest... I don't personally care: Dominika, how about you?

Reply:
I also think that one test should be enough. Hopefully somewhere early in test execution.